### PR TITLE
MM-40861 - Fix for: Untranslatable strings (Playbooks RHS)

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Added translatable strings; added spacing

Before:
image
image

After:
image
image

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-40861

#### Checklist

- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

**Review by calebroseland — APPROVED** [review]
LGTM! 

Though there's one more in this file if it's quick enough https://github.com/mattermost/mattermost-plugin-playbooks/blob/MM-40861-untranslatable-strings/webapp/src/components/rhs/rhs_home.tsx#L331

**Review by abhijit-singh — APPROVED** [review]
Looks good! Based on the screenshots, I believe the text won't overlap with the buttons even if the button text is long so that works 👍

File: webapp/src/components/rhs/rhs_home.tsx [hunk 1]
```diff
@@ -79,20 +81,17 @@ const WelcomeWarn = styled(WelcomeDesc)`
...
-        linear-gradient(
-            180deg,
-            rgba(var(--center-channel-bg-rgb), 0.85) 0%,
-            rgba(var(--center-channel-bg-rgb), 0.25) 100%
-        ),
-        rgba(var(${({exists}) => (exists ? '--button-bg-rgb' : '--center-channel-color-rgb')}), 0.08);
+    background: linear-gradient(180deg,
+    rgba(var(--center-channel-bg-rgb), 0.85) 0%,
+    rgba(var(--center-channel-bg-rgb), 0.25) 100%),
+    rgba(var(${({exists}) => (exists ? '--button-bg-rgb' : '--center-channel-color-rgb')}), 0.08);
```
Comment (agarciamontoro) [inline webapp/src/components/rhs/rhs_home.tsx:94]: I guess this is just your editor changing stuff for you, but I liked the previous format better.

**Review by agarciamontoro — APPROVED** [review]
Neat!

File: webapp/src/components/rhs/rhs_home.tsx (hunk 1 above)
Comment (cpoile) [reply webapp/src/components/rhs/rhs_home.tsx]: Yep, I changed it back. Let's see how long it lasts with the editors doing their thing. :)

**Pull request merged**